examples/java/spring-boot/src/test/resources/application.properties

```diff
@@ -0,0 +1 @@
+fixedRate.in.milliseconds = 500
```

Comment:
also, is there a better name here? related to _what_ this `fixedRate` is actually related to?